src/components/Home/WhatIsSus.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+    <div class="what-is-sus">
+        <div class="text">
+            <h1>What is Startup Summit?</h1>
+            <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.</p>
+            <button class="learn-more" type="button">Learn more</button>
+        </div>
+        <div class="picture"></div>
```

Comment:
Let's add the picture first before merging. Make sure to add an alt = "something"